### PR TITLE
DPL Analysis: Added histogram fill

### PR DESCRIPTION
**Pull request by saganatt**

### CODE REVIEW

File: Framework/Core/include/Framework/HistogramRegistry.h [hunk 1]
```diff
@@ -228,6 +228,13 @@ class HistogramRegistry
     throw std::runtime_error("No match found!");
   }
 
+  template <typename T>
+  void fill(char const* const name, T value)
+  {
+    auto& hist = get(name);
```
Comment (ktf) [inline Framework/Core/include/Framework/HistogramRegistry.h:234]: mmm... I would avoid something like this. Clearly we do not want to encourage people to fill histograms by name..

Comment (saganatt) [reply Framework/Core/include/Framework/HistogramRegistry.h:234]: ok, then what's the preferred option?
Sorry, I don't know what's the convention here ;-)

**Comment by ktf** [comment]
i would simply let them get the histogram and fill manually. if we want to provide an easier way, i would concentrate on using expressions to do so.

**Comment by saganatt** [comment]
> i would simply let them get the histogram and fill manually. if we want to provide an easier way, i would concentrate on using expressions to do so.

ok

**Pull request closed without merging**